lbry/lbry/extras/daemon/Daemon.py

```diff
@@ -1544,15 +1545,32 @@ def jsonrpc_file_list(self, sort=None, reverse=False, comparison=None, **kwargs)
             --blobs_remaining=<blobs_remaining>    : (int) amount of remaining blobs to download
             --sort=<sort_by>                       : (str) field to sort by (one of the above filter fields)
             --comparison=<comparison>              : (str) logical comparision, (eq | ne | g | ge | l | le)
+            --page=<page>                          : (int) page to view within paginated output
+            --page_size=<page_size>                : (int) size of each page within paginated output
 
-        Returns: {List[File]}
+        Returns: {Paginated[Output]}
         """
         sort = sort or 'rowid'
         comparison = comparison or 'eq'
-        return self.stream_manager.get_filtered_streams(
+
+        file_list = self.stream_manager.get_filtered_streams(
             sort, reverse, comparison, **kwargs
         )
 
+        if None not in (page, page_size):
+            total_items = len(file_list)
+            offset = page_size * (page-1)
+            return {
+                'total_items': total_items,
+                'total_pages': int((total_items + (page_size-1)) / page_size),
+                'page': page,
+                'page_size': page_size,
+                'items': file_list[offset:offset+page_size]
+            }
+
+        return file_list
```

Comment:
The method should return the same return structure (either `Paginated` or `List` but not both).

Reply:
I fixed by defaulting `page` and `page_size` to `1` and `10` respectively.